lib/Firtool/Firtool.cpp

```diff
@@ -272,6 +272,7 @@ LogicalResult firtool::populateHWToSV(mlir::PassManager &pm,
        !opt.isRandomEnabled(FirtoolOptions::RandomKind::Mem),
        /*emitSeparateAlwaysBlocks=*/
        opt.shouldEmitSeparateAlwaysBlocks()}));
+  pm.addNestedPass<hw::HWModuleOp>(circt::createLowerSimToSVPass());
   pm.addNestedPass<hw::HWModuleOp>(createLowerVerifToSVPass());
```

Comment:
Consider combining these together. That may allow this to run faster:

```suggestion
  pm.addNestedPass<hw::HWModuleOp>(circt::createLowerSimToSVPass())
    .addNestedPass<hw::HWModuleOp>(createLowerVerifToSVPass());
```

Reply:
Since they are module-level, I think they are already scheduled as a group?